css/styles.css

```diff
@@ -144,31 +192,45 @@ body {
   border: 2px solid #ffffff;
 }
 
-.section__start-screen, .section__pop-up {
+.section__pop-up {
   height: 400px;
   width: 100%;
   position: absolute;
 }
 
 .article__header {
-  text-shadow: 2px 2px #000000;
+  position: relative;
```

Comment:
These are all of the displayed categories on the game board. Blanchedalmond is the obvious choice because it resembles a card! The suits are sometimes running into the category name depending on how long the category name is. Im not sure what the best plan is for that, but i think its a nice touch if we can control the spacing with all the category name lengths.